DEPS.bzl

```diff
@@ -787,10 +787,10 @@ def go_deps():
         name = "com_github_axiomhq_hyperloglog",
         build_file_proto_mode = "disable_global",
         importpath = "github.com/axiomhq/hyperloglog",
-        sha256 = "fdbbf370d3ddb6b5463a88678b9e2d78d31dff46f11e8cbb9a9d58ba8af1af2f",
-        strip_prefix = "github.com/axiomhq/hyperloglog@v0.2.0",
+        sha256 = "812834322ee2ca50dc36f91f9ac3f2cde4631af2f9c330b1271c78b46024a540",
```

Comment:
Please use `6125b12664bb5dd8614e82f0fe7528242dcb11649e1d7e051aabf3da471e14e1` and you should be good to go.